hedera-node/src/test/java/com/hedera/services/tokens/HederaTokenStoreTest.java

```diff
@@ -1225,36 +1219,6 @@ public void happyPathWorks() {
 		assertEquals(expected, subject.pendingCreation);
 	}
 
-	@Test
-	public void rejectsMissingAdminKey() {
-		// given:
-		var req = fullyValidAttempt()
-				.clearAdminKey()
-				.build();
-
-		// when:
-		var result = subject.createProvisionally(req, sponsor);
-
-		// then:
-		assertEquals(INVALID_ADMIN_KEY, result.getStatus());
-		assertTrue(result.getCreated().isEmpty());
-	}
-
-	@Test
-	public void rejectsInvalidAdminKey() {
```

Comment:
I think that this test could be left and not deleted. If on token creation someone passes invalid admin key, we should tell that it is an invalid one right?